### PR TITLE
[17.0][MIG] report_xlsx: Migration to 17.0

### PR DESCRIPTION
**Pull request by tien-ld**

### CODE REVIEW

File: report_xlsx/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "author": "ACSONE SA/NV," "Creu Blanca," "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/reporting-engine",
     "category": "Reporting",
-    "version": "16.0.2.0.0",
+    "version": "17.0.2.0.0",
```
Comment (atchuthan) [inline report_xlsx/__manifest__.py]: 17.0.1.0.0

File: report_xlsx/models/ir_report.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def _render_xlsx(self, report_ref, docids, data):
             .sudo(False)
             .create_xlsx_report(docids, data)  # noqa
         )
-        if ret and isinstance(ret, (tuple, list)):  # data, "xlsx"
+        if ret and isinstance(ret, (tuple | list)):  # data, "xlsx"
```
Comment (atchuthan) [inline report_xlsx/models/ir_report.py:29]: Why did you use bitwise OR?

**Review by atchuthan — CHANGES_REQUESTED** [review]
Except for minor changes, code looks good :+1: 
Tested in local instance functionally.

**Review by gurneyalex — APPROVED** [review]

File: report_xlsx/models/ir_report.py (hunk 2 above)
Comment (tien-ld) [reply report_xlsx/models/ir_report.py:29]: With python 3.10, we should use `|`, you can find more info here: https://peps.python.org/pep-0604/

File: report_xlsx/__manifest__.py (hunk 1 above)
Comment (gurneyalex) [reply report_xlsx/__manifest__.py]: I fixed this

**Comment by sandalydiallo** [comment]
Hello, does anyone know when this module will be published on odoo apps?

**Pull request merged**